org/ystia/yorc/yorc/linux/ansible/playbooks/templates/config.yorc.consul.yaml.j2

```diff
@@ -0,0 +1,3 @@
+consul:
+  address: "{{CONSUL_IP}}:{{CONSUL_PORT}}"
+http_port: "{{REST_API_PORT}}"
```

Comment:
This is not related to consul and should be in a dedicated file or in org/ystia/yorc/yorc/linux/ansible/playbooks/templates/config.yorc.yaml.j2